resolver/resolver.go

```diff
@@ -1002,6 +967,109 @@ func (r *Resolver) validateMessageFields(ctx *context, msg *Message, builder *so
 	}
 }
 
+func (r *Resolver) validateRequestFieldType(ctx *context, fromType *Type, toField *Field, builder *source.RequestOptionBuilder) {
+	if fromType == nil || toField == nil {
+		return
+	}
+	toType := toField.Type
+	if toType.Kind == types.Message {
```

Comment:
Could toType be nil here?

Reply:
It's OK, `toField` is one of the target gRPC method request message's field, so it is not nil always.